src/main/java/uk/gov/justice/hmpps/prison/repository/jpa/repository/OffenderAlertRepository.java

```diff
@@ -1,11 +1,16 @@
 package uk.gov.justice.hmpps.prison.repository.jpa.repository;
 
 import org.springframework.data.jpa.repository.JpaSpecificationExecutor;
+import org.springframework.data.repository.CrudRepository;
 import org.springframework.data.repository.PagingAndSortingRepository;
 import uk.gov.justice.hmpps.prison.repository.jpa.model.OffenderAlert;
 
 import java.util.List;
 
-public interface OffenderAlertRepository extends PagingAndSortingRepository<OffenderAlert, OffenderAlert.PK>, JpaSpecificationExecutor<OffenderAlert> {
+public interface OffenderAlertRepository extends
+    PagingAndSortingRepository<OffenderAlert, OffenderAlert.PK>,
+    CrudRepository<OffenderAlert, OffenderAlert.PK>,
```

Comment:
`CrudRepository` isn't now automatically included if `PagingAndSortingRepository` specified, so needs to be included manually